restfull-api/controllers/todo.js

```diff
@@ -5,7 +5,16 @@ class TodoController {
   }
 
   async getAllTodos(req, res) {
-    const todos = await this.Todo.find();
+    const page = req.query.page || 0;
+    const todoPerPage = 3;
+    const isDoneQuery = { isDone: false };
+    const notDeletedQuery = { deletedAt: null };
+    const isVisible = isDoneQuery && notDeletedQuery;
```

Comment:
Can you explain what you want to achieve here?

Reply:
line 9: I assume that in each page app going to show 3 last todo list
line 10:  is done  todos not going to be shown 
line 11: deleted todos not going to be shown
line 12: logical &

Reply:
log line 12 and check the result